file-formats/srvc/type/zif_aff_srvc_v1.intf.abap

```diff
@@ -34,42 +38,41 @@ INTERFACE zif_aff_srvc_v1
     END OF ty_connection_property.
 
   "! <p class="shorttext">Connection Properties</p>
-  "! Connection Properties
+  "! Connection properties
   TYPES ty_connection_properties TYPE TABLE OF ty_connection_property WITH DEFAULT KEY.
 
   TYPES:
     "! <p class="shorttext">Consumer Information</p>
-    "! Consumer Information
-    BEGIN OF ty_consumer_information,
-      "! <p class="shorttext">Consumer type</p>
-      "! Consumer type
-      consumer_type         TYPE ty_cons_type,
-      "! <p class="shorttext">Consumer object name</p>
-      "! Consumer object name
-      consumer_object_name  TYPE c LENGTH 40,
-      "! <p class="shorttext">Consumer object type</p>
-      "! Consumer object type
-      consumer_object_type  TYPE c LENGTH 4,
-      "! <p class="shorttext">Connection properties</p>
-      "! Connection properties
-      "! $required
-      connection_properties TYPE ty_connection_properties,
-    END OF ty_consumer_information.
+    "! Consumer information
+    BEGIN OF ty_consumer,
+      "! <p class="shorttext">Type</p>
+      "! Type
+      type        TYPE ty_cons_type,
+      "! <p class="shorttext">Object Type</p>
+      "! Object type
+      object_type TYPE c LENGTH 4,
+      "! <p class="shorttext">Object Name</p>
+      "! Object name
+      object_name TYPE c LENGTH 40,
+      "! <p class="shorttext">Properties</p>
+      "! Properties
+      properties  TYPE ty_connection_properties,
+    END OF ty_consumer.
 
   TYPES:
     "! <p class="shorttext">Service Consumption Model Properties</p>
     "! Service Consumption Model properties
     BEGIN OF ty_main,
       "! $required
-      format_version       TYPE zif_aff_types_v1=>ty_format_version,
+      format_version TYPE zif_aff_types_v1=>ty_format_version,
       "! <p class="shorttext">Header</p>
       "! Header
       "! $required
-      header               TYPE zif_aff_types_v1=>ty_header_60,
-      "! <p class="shorttext">Consumer information</p>
-      "! Consumer information
+      header         TYPE zif_aff_types_v1=>ty_header_60,
```

Comment:
If SVRC does not support key user extensibility, you can change the type to `ty_header_60_cloud`.

```suggestion
      header         TYPE zif_aff_types_v1=>ty_header_60_cloud,
```